packages/blockchain-extension/test/commands/packageSmartContractCommand.test.ts

```diff
@@ -809,7 +809,9 @@ describe('packageSmartContract', () => {
             logSpy.should.have.been.calledTwice;
         });
 
-        it('should throw an error if the JavaScript project already exists', async () => {
+        it('should allow the user to replace the existing smart contract package', async () => {
+            const warningStub: sinon.SinonStub = mySandBox.stub(vscode.window, 'showWarningMessage').resolves({ title: 'Continue (replace)' });
```

Comment:
Same thing here - you're resolving `Continue (replace)` which is not what the user can actually pick (Yes or No).